mmdet/models/losses/dice_loss.py

```diff
@@ -121,3 +121,73 @@ def forward(self,
             avg_factor=avg_factor)
 
         return loss
+
+
+def naive_dice_loss(pred,
+                    target,
+                    weight=None,
+                    eps=1e-3,
+                    reduction='mean',
+                    avg_factor=None):
+
+    input = pred.flatten(1)
+    target = target.flatten(1).float()
+
+    a = torch.sum(input * target, 1)
+    b = torch.sum(input, 1)
+    c = torch.sum(target, 1)
+    d = (2 * a + eps) / (b + c + eps)
+    loss = 1 - d
+    if weight is not None:
+        assert weight.ndim == loss.ndim
+        assert len(weight) == len(pred)
+    loss = weight_reduce_loss(loss, weight, reduction, avg_factor)
+    return loss
+
+
+@LOSSES.register_module()
+class NaiveDiceLoss(DiceLoss):
```

Comment:
It is necessary to write a new loss, you can add an argument to the original function to change the calculation mode.